.github/workflows/log-viewer-docs.yaml

```
@@ -2,7 +2,15 @@
 
 on:
   pull_request:
+    paths: &docs_paths
+      - "docs/**"
+      - ".github/workflows/log-viewer-docs.yaml"
+      - ".gitmodules"
+      - "taskfile.yaml"
+      - "tools/scripts/**"
+      - "tools/yscope-dev-utils"
   push:
+    paths: *docs_paths
```

actionlint (1.7.7)

"paths" section must be sequence node but got alias node with "" tag

(syntax-check)